screens/wait_scan.py

```diff
@@ -148,10 +147,10 @@ def stat_drink(self, args, pos):
             DrinksManager.get_instance().set_selected_drink(None)
         self.reset()
 
-    def btn_reset(self, args, pos):
+    def btn_reset(self):
         self.reset()
 
-    def btn_new_id(self, args, pos):
+    def btn_new_id(self):
```

Comment:
see above... and more places

sounds like we need an abstract class to enforce typings... or can we type click_func?